zsl/cache/redis_cache_module.py

```diff
@@ -20,12 +20,13 @@ def __init__(self, app, config):
         self._app = app
         self._config = config
 
-        redis_conf = self._config['REDIS']
+        redis_conf = self._config.get('REDIS', {})
```

Comment:
Maybe we should test for a complete configuration and display a warning or error.